frontend/views/containers/chatroom/Emoticons.vue

```diff
@@ -89,8 +89,15 @@ export default ({
       }
     },
     closeEmoticonDlg () {
+      if (!this.isActive) { return }
+
       this.isActive = false
       sbp('okTurtles.events/emit', CLOSE_EMOTICON)
+
+      if (this.lastFocus) {
+        this.lastFocus.focus()
+        this.lastFocus = null
+      }
```

Comment:
(This is maybe not a big deal but) I also noticed that the component captures the lastly focused element when the emoticon pop-up opens(in `openEmoticon()` method above in the file) but that is not used anywhere. So added this logic here too.